chromium-headless/start_chrome_headless.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+for value in {1..3}
+do
+    echo "Trying to start chrome, attempt $value"
+    ps -aux | grep "chrome" | grep "9222" > /dev/null 2>&1
+    if [[ "$?" == "0" ]]; then
+        echo "Chrome is running"
+        exit 0
+    else
+        echo "Chrome is not running"
+        service dbus restart
+        nohup /opt/google/chrome/google-chrome --headless --disable-gpu --no-sandbox --hide-scrollbars --disable_infobars --start-maximized --start-fullscreen --ignore-certificate-errors --disable-dev-shm-usage --user-agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36" --remote-debugging-port=9222 & disown
```

Comment:
does the user agent matter? probably isnt macintosh

Reply:
you sure we dont want this script to be implemented on the integration side? --ignore-certificate-errors for example would make sense to me to be dependant on verify=False.

Reply:
> does the user agent matter? probably isnt macintosh

That's the value we use today.

> ignore-certificate-errors

That's how it is now.
the `rasterize` integration doesn't have this option.